### PR TITLE
feat(html): unify Tooltip rendering

### PR DESCRIPTION
**Pull request by inikolova**
**!!! Do not merge** (until 5th Decemeber) !!!

Related to https://github.com/telerik/kendo-themes-private/issues/187

**Note**: Tooltip elevation level has been changed after sync with @dmanova and follows the update design guidelines - https://www.figma.com/file/r0H47iKFdkELP2bpX6hU9c/Elevation-System?type=design&node-id=269-5730&mode=design&t=N1ztFaCuVVcLDtPk-0

**Note**: Validation tooltips have shadow now - we discussed it in [this meeting](https://github.com/telerik/web-components-ux/discussions/955#discussioncomment-7640310) and agreed that validation tooltips should have the same shadow as the tooltip component.

### CODE REVIEW

File: packages/default/scss/tooltip/_theme.scss [hunk 1]
```diff
@@ -1,80 +1,31 @@
...
-
     // Default tooltip
     .k-tooltip {
         @include fill(
             $kendo-tooltip-text,
             $kendo-tooltip-bg,
             $kendo-tooltip-border
         );
+        
+        box-shadow: $kendo-tooltip-shadow;
```
Comment (epetrow) [inline packages/default/scss/tooltip/_theme.scss:10]: We can maybe use the box-shadow mixin for consistency here e.g. @include box-shadow( $kendo-tooltip-shadow ) ?

File: packages/html/src/tooltip/tests/tooltip.tsx
Comment (epetrow) [inline packages/html/src/tooltip/tests/tooltip.tsx:1]: If we are unifying the rendering do we need to include expamples for the different suites in this test?

**Review by epetrow — COMMENTED** [review]
Also maybe the tooltip test could be made in a way similar to [this one](https://github.com/telerik/kendo-themes/blob/develop/tests/list/list-grouping.html) if there are suite specifics that we cannot escape?

File: packages/html/src/tooltip/tests/tooltip-callout-position.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    .k-tooltip {
+        position: relative;
+    }
+`;
+
+
+export default () =>(
+    <>
+        <style>{style}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-5">
```
Comment (zhpenkov) [inline packages/html/src/tooltip/tests/tooltip-callout-position.tsx:24]:
```suggestion
        <div id="test-area" className="k-d-grid">
```
Those classNames become redundant after applying this **#test-area** style
```css
    #test-area {
        grid-template-columns: 120px repeat(4, 1fr);
    }
``` 
(appears in **tooltip.tsx** and **tooltip-rtl.tsx** as well)

File: packages/html/src/tooltip/tests/tooltip-callout-position.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal title="Tooltip title" callout="left" />
+                </AnimationContainer>
+            </section>
+
+            <span>Long content & Title (with glyphs)</span>
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal content="long tooltip content. Enough so it goes on another line" title="g y p q f t d l" callout="bottom" />
```
Comment (zhpenkov) [inline packages/html/src/tooltip/tests/tooltip-callout-position.tsx:110]: This is purely a cosmetic suggestion.
```suggestion
                    <TooltipNormal content="Long tooltip content. Enough so it goes on another line" title="g y p q f t d l" callout="bottom" />
```
Some long content sentences begin with uppercase L, others with lowercase L.
(appears in **tooltip.tsx** and **tooltip-rtl.tsx** as well)

**Review by epetrow — APPROVED** [review]

**Review by zhpenkov — APPROVED** [review]

**Review by JoomFX — APPROVED** [review]

**Pull request merged**